src/InteractiveWindow/EditorTest/app.config

```diff
@@ -13,4 +13,12 @@
       </listeners>
     </trace>
   </system.diagnostics>
+  <runtime>
+    <assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1">
+      <dependentAssembly>
+        <assemblyIdentity name="System.Collections.Immutable" publicKeyToken="b03f5f7f11d50a3a" culture="neutral"/>
+        <bindingRedirect oldVersion="0.0.0.0-1.2.0.0" newVersion="1.2.0.0"/>
```

Comment:
Can we generate this dynamically instead? This might just come for free if we flip on the switches @Pilchie added.